pkg/pillar/containerd/containerd.go

```diff
@@ -634,6 +634,7 @@ func LKTaskPrepare(name, linuxkit string, domSettings *types.DomainConfig, memOv
 		if memOverhead > 0 {
 			spec.AdjustMemLimit(*domSettings, memOverhead)
 		}
+		spec.UpdateMounts(domSettings.DiskConfigList)
```

Comment:
I'm wondering if you need to do the translation of where things get mounted here. Unlike bare metal container, this is going to be applied to, what effectively is xen-tools -- hence what you really need is not to mount within that namespace but rather within rootfs namespace, no? @zed-rishabh

Reply:
In my knowledge, we can add mounts while creating container in containerd world but I am not sure how can we add mounts while preparing snapshots.